BlockMap-internal/src/main/resources/block-color-instructions-1_17.json

```diff
@@ -573,7 +573,153 @@
+
+			"deepslate": "deepslate.png",
+			"deepslate_bricks": "deepslate_bricks.png",
+			"deepslate_brick_slab": "deepslate_bricks.png",
+			"deepslate_brick_stairs": "deepslate_bricks.png",
+			"deepslate_brick_wall": "deepslate_bricks.png",
+			"deepslate_coal_ore": "deepslate_coal_ore.png",
+			"deepslate_copper_ore": "deepslate_copper_ore.png",
+			"deepslate_diamond_ore": "deepslate_diamond_ore.png",
+			"deepslate_emerald_ore": "deepslate_emerald_ore.png",
+			"deepslate_gold_ore": "deepslate_gold_ore.png",
+			"deepslate_iron_ore": "deepslate_iron_ore.png",
+			"deepslate_lapis_ore": "deepslate_lapis_ore.png",
+			"deepslate_redstone_ore": "deepslate_redstone_ore.png",
+			"deepslate_tiles": "deepslate_tiles.png",
+			"deepslate_tile_slab": "deepslate_tiles.png",
+			"deepslate_tile_stairs": "deepslate_tiles.png",
+			"deepslate_tile_wall": "deepslate_tiles.png",
+			"infested_deepslate": "deepslate.png",
+			"polished_deepslate": "polished_deepslate.png",
+			"polished_deepslate_slab": "polished_deepslate.png",
+			"polished_deepslate_stairs": "polished_deepslate.png",
+			"polished_deepslate_wall": "polished_deepslate.png",
+			"chiseled_deepslate": "chiseled_deepslate.png",
+			"cracked_deepslate_bricks": "cracked_deepslate_bricks.png",
+			"cracked_deepslate_tiles": "cracked_deepslate_tiles.png",
+			"cobbled_deepslate": "cobbled_deepslate.png",
+			"cobbled_deepslate_slab": "cobbled_deepslate.png",
+			"cobbled_deepslate_stairs": "cobbled_deepslate.png",
+			"cobbled_deepslate_wall": "cobbled_deepslate.png",
+
+			"raw_copper_block": "raw_copper_block.png",
+			"raw_iron_block": "raw_iron_block.png",
+			"raw_gold_block": "raw_gold_block.png",
+
+			"azalea": "azalea_top.png",
+			"azalea_leaves": "azalea_leaves.png",
+			"flowering_azalea": "flowering_azalea_top.png",
+			"flowering_azalea_leaves": "flowering_azalea_leaves.png",
+			"potted_azalea_bush": "potted_azalea_bush_top.png",
+			"potted_flowering_azalea_bush": "potted_flowering_azalea_bush_top.png",
+
+			"amethyst_block": "amethyst_block.png",
+			"amethyst_cluster": "amethyst_cluster.png",
+			"budding_amethyst": "budding_amethyst.png",
+			"large_amethyst_bud": "large_amethyst_bud.png",
+			"medium_amethyst_bud": "medium_amethyst_bud.png",
+			"small_amethyst_bud": "small_amethyst_bud.png",
+
+			"big_dripleaf": "big_dripleaf_top.png",
+			"big_dripleaf_stem": "big_dripleaf_stem.png",
+			"small_dripleaf": "small_dripleaf_top.png",
+
+			"cauldron": "cauldron_top.png",
+			"lava_cauldron": "cauldron_top.png",
+			"powder_snow_cauldron": "cauldron_top.png",
+			"water_cauldron": "cauldron_top.png",
+
+			"calcite": "calcite.png",
+			"cave_vines": "cave_vines.png",
+			"cave_vines_plant": "cave_vines_plant.png",
+			"copper_ore": "copper_ore.png",
+			"dirt_path": "dirt.png",
+			"dripstone_block": "dripstone_block.png",
+			"glow_lichen": "glow_lichen.png",
+			"hanging_roots": "hanging_roots.png",
+			"light":  "white_candle.png",
```

Comment:
Could not find a texture for the Light Block

Reply:
You can use `["fixed", "0xFFA26B23"]` to give it the color you want, or `["transparent"]` if it shouldn't be part of the map.